requirements.txt

```diff
@@ -46,13 +46,12 @@ python-gdcm==3.0.20
 pytz==2022.7
 PyWavelets==1.4.1
 requests==2.28.1
-rt-utils==1.2.7
+-e git+https://github.com/qurit/rt-utils@769acd6#egg=rt-utils
```

Comment:
This is currently not working – apparently installs, but `ModuleNotFound` error on execution of `import rt_utils` in Python.

Will have to implement workaround until rt-utils PyPI package updated.